api/src/common/decorators_api_views.py

```diff
@@ -0,0 +1,18 @@
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+
+# common decorators applied to all model view set classes
+def viewsets(tag):
+    print ('tag name is: ' + tag)
+
```

Comment:
We should remove all print statements before merging to main.

Reply:
done